src/resources/help/en_US/relnotes3.2.0.html

```diff
@@ -116,6 +116,7 @@
   <li>Removed FastBit and FastQuery from VisIt.</li>
   <li>Fixed a bug with build_visit building IceT with MPICH.</li>
   <li>Fixed a bug with build_visit building IceT on a system without OpenGL installed.</li>
+  <li>Fixed a bug that could cause LibSim examples to crash.</li>
```

Comment:
I don't think this is the correct spot for this. Maybe put it in "Bugs fixed" section. Maybe also change "could cause LibSim " to "caused some LibSim".

Reply:
Good catch, thanks!. Fixed.